packages/noir-ethereum-api/test/e2e.test.ts

```diff
@@ -28,6 +35,33 @@ describe.concurrent(
       expect(await verifyStorageProof(proofData)).toEqual(true);
     });
 
+    it('anvil smart contract proof verification successes', async () => {
+      const client = createAnvilClient();
+
+      const account: Account = privateKeyToAccount(ANVIL_TEST_ACCOUNT_PRIVATE_KEY);
+
+      const contract: Hex = await client.deployContract({
+        abi: ultraVerifier.abi,
+        account,
+        bytecode: ultraVerifier.bytecode.object as Hex
+      });
+
+      const transaction: TransactionReceipt = await client.waitForTransactionReceipt({ hash: contract });
+      expect(transaction.status).toEqual('success');
+
+      const contractAddress = transaction.contractAddress as Hex;
```

Comment:
Just:
```ts
const {contractAddress} = transaction.contractAddress;
```

`contractAddress` is of type `0x${string}` same as Hex, compiler should infer